src/main/java/com/moodybluez/enterprise/MoodyBluezController.java

```diff
@@ -44,7 +48,7 @@ public String processRegister(User user) throws Exception {
 
         userService.save(user);
 
-        return "index";
+        return "signup";
```

Comment:
Since I moved the signup page, I changed the page that was returned after signing up a new user. The program spits the password out on the screen with the form.